src/Moryx.ControlSystem/VisualInstructions/InstructionResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Moryx.ControlSystem.VisualInstructions
+{
+    /// <summary>
+    /// Object representing a possible result for an instructions
+    /// </summary>
+    public class InstructionResult
+    {
```

Comment:
xml comments for public properties

Key of the result. This value needs to be unique for the instruction the result is used for.

Human readable value of the result